doc/telescope.txt

```diff
@@ -116,22 +116,6 @@ builtin.live_grep()                                      *builtin.live_grep()*
 
 Actions functions that are useful for people creating their own mappings.
 
-actions.move_selection_next({prompt_bufnr})    *actions.move_selection_next()*
```

Comment:
I had this same issue in #759 @tjdevries, because of the max query bug with treesitter this section of docs is getting deleted by the docgen